src/app/content/search/reducer.ts

```diff
@@ -27,6 +28,9 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
         ? initialState
         : state;
     }
+    case getType(actions.toggleSearchSidebar): {
+      return {...state, open: action.payload};
```

Comment:
if this is a toggle then it should be `return {...state, open: !state.open};`, if you need to be able to pass in the open/closed boolean to make something work then probably should create two actions `openResults` and `closeResults`